sdk/program/src/program.rs

```diff
@@ -35,6 +49,19 @@ pub fn invoke_signed(
         }
     }
 
+    invoke_signed_unchecked(instruction, account_infos, signers_seeds)
+}
+
+/// Invoke a cross-program instruction with program signatures but don't check
+/// RefCell handling.
```

Comment:
Can you also add something like:
> These checks ensure that the invocation doesn't violate the borrow rules of the `AccountInfo` fields that are wrapped in `RefCell`'s

Reply:
Just so that people know why it's valuable to have those checks and they take on risk if they skip them